### PR TITLE
Bug #1276071 - Utility sidebar content 3 - Customization (themes)

### PR DESCRIPTION
**Pull request by FrancescoSTL**
This should reflect the initial commit for the third sidebar; customization.

### CODE REVIEW

File: data/js/content3.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+(function() {
+    // retrieve our buttons from the page
+    var tbutton1 = document.querySelector('#template1');
+    var tbutton2 = document.querySelector('#template2');
+    var tbutton3 = document.querySelector('#template3');
+    var defaultText = document.querySelector('#default');
+    var timeout;
+
+    // add event listeners for a button click on all three buttons
+    tbutton1.addEventListener('click', function() {
```
Comment (FrancescoSTL) [inline data/js/content3.js]: These event listeners should eventually be put into an array but I'm currently running into an issue with modifying any button properties in a loop.

**Comment by FrancescoSTL** [comment]
Todo for this branch:
1. Replace SVG icons that mternoway sent and send svgs through utility
2. Fix folder higherarchy
3. Replace titles
4. Add "default" to bottom of switch statements
5. Rebase with master

**Comment by schalkneethling** [comment]
This one still needs a rebase.

**Comment by FrancescoSTL** [comment]
Branch ready for merge.

**Comment by schalkneethling** [comment]
There is still some work required here but, to move this forward and get all the code up to the main Mozilla repo, I am merging these as is.

**Pull request merged**